### DISCUSSION TITLE
A page requires the use of two VueFlows panel

### DISCUSSION THREAD

**Discussion by yangsongtao1** [bcakmakoglu/vue-flow#1442] [Q&A]
How to use two VueFlows on a page with independent data

I use different data assignments, but the data displayed on the panel is always the same

Data:
![data](https://github.com/bcakmakoglu/vue-flow/assets/51110190/1c5d5eda-9218-47ab-ba7e-231b95c90e1f)

Template:
![template](https://github.com/bcakmakoglu/vue-flow/assets/51110190/e797dc70-daad-4b0e-950d-e182835c4d18)

Result:
![结果](https://github.com/bcakmakoglu/vue-flow/assets/51110190/41e02cba-a512-43f9-b8e5-2ec3b188efd2)

**Comment by bcakmakoglu** [comment]
Could you please provide a reproduction of this issue.
[You can find a template here.](https://codesandbox.io/p/devbox/vue-flow-basic-gfgro4)
